src/components/inputs/Checkbox/Checkbox.scss

```diff
@@ -0,0 +1,76 @@
+$C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-checkbox;
+
+.#{$C} {
+  display: flex;
+  justify-content: center;
```

Comment:
I think this should be left, otherwise it will be difficult to align multiple checkboxes